help/implementation-playbook/best-practices/planning/redis-service-configuration.md

```diff
@@ -46,6 +46,13 @@ For details, see [Deploy variables > `REDIS_BACKEND`](https://devdocs.magento.co
 >
 > Check the ece-tools version installed in your local Cloud environment from the command line using the `composer show magento/ece-tools` command. If necessary, [update the ece-tools version](https://devdocs.magento.com/cloud/project/ece-tools-update.html). 
 
+</br>
+
+>[!WARNING]
+>
+>For scaled architecture (split architecture), Redis slave connections **SHOULD NOT** be enabled. You can check if you are on scaled architecture by going to your project URL, e.g. https:&#8203;//us.magento.cloud/projects/&lt;project ID&gt;/environments/production. Click on the Access site. If there are more than three nodes shown under SSH access, you are on scaled architecture. If you enable Redis Slave Reads on scaled architecture, the customer will receive errors on Redis connections not being able to connect. This has to do with how the clusters are configured to process Redis connections. Redis Slaves are still active but will not be used for Redis Reads. We recommend for scaled architecture to use Adobe Commerce 2.3.5 or later and implement new Redis back-end configuration and implement L2 caching for Redis.
```

Comment:
Is this expected?  https:&#8203;//us.magento

Reply:
Yeah, that's weird. I'd prefer to see the encoding removed and then the URL wrapped with inline code formatting:

For example:

`https://us.magento.cloud/projects/<project ID>/environments/production`

Reply:
This is an altered copy of the [warning from the Cloud guide](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure/env/stage/variables-deploy.html?lang=en#redis_use_slave_connection). I think it would be best to make this callout brief and include a link to the Cloud guide configuration option for the Redis configuration variable in question.

Reply:
I copied over and pasted, so I can see formatting got a bit off sync, thanks for highlighting.